draft-packages/form/KaizenDraft/Form/Primitives/FieldMessage/styles.scss

```diff
@@ -4,30 +4,49 @@
 @import "~@kaizen/deprecated-component-library-helpers/styles/type";
 @import "../../decision-tokens.scss";
 
+$message-icon-size: 1.25rem;
+
 .message {
   display: flex;
+  position: relative;
   align-items: center;
-  gap: $ca-grid * 0.25;
+  gap: $spacing-xs;
 }
 
 .default {
   color: $dt-color-form-text-color-label;
+
   &.reversed {
     color: $dt-color-form-text-color-label-reversed;
   }
 }
 
-.error {
-  padding: $ca-grid / 2;
-  color: $dt-color-form-text-color;
+.error,
+.caution {
+  border: $border-solid-border-width $border-solid-border-style transparent;
```

Comment:
Sidenote: Not a big fan of using the `border` shorthand - In this particular scenario I have a hard time understanding what attribute `transparent` sets. There's also a single `border-radius` just below.
I'd find it more readable to be more verbose here, write out `border-width... border-style...`